src/parse.rs

```diff
@@ -2065,10 +2065,10 @@ where
                             value: Box::new(value),
                             short_form: false,
                         })),
-                        _ => Ok(Some(BitStringSegmentOption::Invalid {
-                            label: name,
-                            location: SrcSpan { start, end },
-                        })),
```

Comment:
This was the remaining code using the `Invalid` variant. The code below produces a parse error now.